webpack.dev.config.js

```diff
@@ -4,16 +4,23 @@ var common    = require('./webpack.common.config');
 var plugins   = require('./buildtools/webpack/plugins');
 var usePolyfill = require('./buildtools/webpack/polyfill');
 
+var path          = require('path');
+const PLAYGROUND = path.resolve(__dirname, 'playground');
+
+
 module.exports = (env = {}) => {
   const { isProduction, skipAnalyzer } = env;
 
   const webpackConfig = {
     ...common('okta-sign-in.js'),
-    plugins: plugins({ 
+    plugins: plugins({
       isProduction,
       skipAnalyzer,
     })
   };
   usePolyfill(webpackConfig);
+  Object.assign(webpackConfig.resolve.alias, {
+    'duo': `${PLAYGROUND}/mocks/thirdparty/duo-mock.js`,
```

Comment:
seems not ideal? but wasn't able to think of any alternatives :(

Reply:
I'm not sure this is the best idea, as we may not want to **always** use this mock when running in development.

What do we suggest for developers who want to test this against an actual Duo integration?

Reply:
Added `yarn start:mockDuo` to fix this. Take a look